PracticeMode.cs

```diff
@@ -771,11 +771,11 @@ public void OnPracCommand(CCSPlayerController? player, CommandInfo? command)
                 return;
             }
 	    
-			// if (isPractice)
-            // {
-            //     StartMatchMode();
-            //     return;
-            // }
+			if (isPractice)
+            {
+                //StartMatchMode();
+                return;
```

Comment:
Hey!

Thanks for the PR! Though I remember some people want !prac command to re-start the prac, hence this check was commented. It also prints all the help commands again if used, hence I don't think we should return from here

Reply:
not better using !rr for this?